src/components/Encryption.vue

```diff
@@ -30,7 +30,10 @@
 			<!--eslint-enable-->
 		</div>
 		<NcActions v-if="$is_admin && !isEditing">
-			<NcActionButton icon="icon-rename" @click.stop.prevent="openEditFullDiskEncryptionForm" />
+			<NcActionButton icon="icon-rename"
+				:aria-label="t('privacy', 'Adapt encryption')"
+				:title="t('privacy', 'Adapt encryption')"
```

Comment:
This should be upstreamed to the vue lib, so it automatically is done for all other usages of the component?

Reply:
was already done some time ago https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcActions/NcActions.vue#L940-L942

Reply:
AH okay. Maybe we should enforce them at some point then.